src/pages/MirrorPage.tsx

```diff
@@ -0,0 +1,252 @@
+import { stringify } from '@medplum/core';
+import { Resource, OperationOutcome, Bundle, Questionnaire } from '@medplum/fhirtypes';
+import {
+  Button,
+  DefaultResourceTimeline,
+  Document,
+  EncounterTimeline,
+  ErrorBoundary,
+  Form,
+  Loading,
+  MedplumLink,
+  PatientTimeline,
+  ResourceBlame,
+  ResourceForm,
+  ResourceHistoryTable,
+  ResourceTable,
+  ServiceRequestTimeline,
+  Tab,
+  TabList,
+  TabPanel,
+  TabSwitch,
+  TextArea,
+  useMedplum,
+} from '@medplum/react';
+import React, { useState, useCallback, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import './MirrorPage.css';
+import { PatientHeader } from './PatientHeader';
+import { ResourceHeader } from './ResourceHeader';
+import { getPatient } from './utils';
+
+export function getTabs(resourceType: string): string[] {
+  const result = ['Timeline'];
+  result.push('Details', 'Edit', 'History', 'Blame', 'JSON');
+  if (resourceType === 'Bot') {
```

Comment:
If these `if` statements are exclusive of each other, I would turn this into a map/dictionary